ts/Extensions/PriceIndication.ts

```diff
@@ -108,6 +147,68 @@ declare global {
  * @apioption plotOptions.series.lastPrice.enabled
```

Comment:
If all options for the lastPrice are available from the v7.0.0, we might add that information to the `enabled` option.